.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ $coverage = 1 ]]; then mkdir -p build/logs build/cov; fi
   - if [[ $coverage = 1 ]]; then wget https://phar.phpunit.de/phpcov.phar; fi
   - if [[ $coverage = 1 ]]; then wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar; fi
+  - if [[ $lint = 1 ]]; then wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v1.12.2/php-cs-fixer.phar; fi
```

Comment:
Because, for instance, we will not catch the issue with the removal of extra dots with the current setup. And it's a very common problem (I already has it a lot of time while hacking on Symfony).
It will be hard to have the same version on the dev box and on Travis too.

Semver is not relevant here because we don't use the internal API for the tool (only the config file). It's the same thing for phpcov btw.

Anyway installing it through Composer (globally) instead of downloading the phar allows to download the last version of the 1.x branch.